src/browser/commands.js

```diff
@@ -0,0 +1,39 @@
+import { sendKeys, sendMouse } from '@web/test-runner-commands';
+
+function getElementPosition(elem) {
+	const { x, y, width, height } = elem.getBoundingClientRect();
+	return [
+		Math.floor(x + window.scrollX + width / 2),
+		Math.floor(y + window.scrollY + height / 2),
+	];
+}
+
+export async function click(elem) {
+	await sendMouse({ type: 'click', position: getElementPosition(elem) });
+}
+
+export async function clickAt(x, y) {
+	await sendMouse({ type: 'click', position: [x, y] });
+}
+
+export async function focus(elem) {
+	await sendKeys({ press: 'Shift' }); // Tab moves focus, Escape causes dismissible things to close
+	elem.focus({ focusVisible: true });
+}
+
+export async function hover(elem) {
+	await sendMouse({ type: 'move', position: getElementPosition(elem) });
+}
+
+export async function hoverAt(x, y) {
+	await sendMouse({ type: 'move', position: [x, y] });
+}
+
+export async function keyboard(action, keys, elem) {
```

Comment:
Interested in what folks think about the name here. We can also go with `sendKeys`, but since we've ditch the `@web/test-runner-commands` syntax for the rest it didn't seem necessary to stick with it here.